test/common/http/conn_manager_impl_test_base.h

```diff
@@ -28,10 +28,10 @@ using testing::NiceMock;
 namespace Envoy {
 namespace Http {
 
-class HttpConnectionManagerImplTest : public testing::Test, public ConnectionManagerConfig {
+class HttpConnectionManagerImplMixin : public ConnectionManagerConfig {
```

Comment:
Can you document what this mixin is useful for?

Reply:
done

Reply:
What is the reason someone would use this mixin? I think that's the non-obvious aspect that would be helpful to have in the comment.

Reply:
done. 
suggest something if you think it's still not clear.